src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It would be good to add a fallback font in case 'Roboto' doesn't load. For example, you can use a generic font family like 'sans-serif'. This would look like: `font-family: Roboto, sans-serif;`